src/main/webapp/app/shared/date-time-picker/date-time-picker.component.ts

```diff
@@ -66,6 +68,14 @@ export class FormDateTimePickerComponent implements ControlValueAccessor {
         }
     }
 
+    /**
+     * validates a date object
+     * @param date to be validated
+     */
+    isValidDate(date: any) {
```

Comment:
can we avoid any here?

Reply:
I now replaced it with a more concrete type.